packages/bpk-component-chip/src/BpkDismissibleChip.js

```diff
@@ -34,18 +35,31 @@ import {
   COMMON_DEFAULT_PROPS,
   type CommonProps,
 } from './commonTypes';
+import STYLES from './BpkSelectableChip.module.scss';
 
 export type Props = CommonProps;
 
+const getClassName = cssModules(STYLES);
+
 const BpkDismissibleChip = (props: Props) => {
-  const { ...rest } = props;
+  const { className, type, ...rest } = props;
+  const iconClassNames = getClassName(
+    `bpk-chip--${type}-dismissible__trailing-accessory-view`,
+  );
+  const dismissibleClassNames = getClassName(
```

Comment:
very small thing here (sorry about that) could you just name this `classNames` given it's the classes applied to the component of this page so it would be obvious it's dismissible? 😄 Just to be consistent with what we do in all our other components

Reply:
Ok, very good advice, it is important to follow the development specifications. Will fix this soon. 😄